docs/tutorials/exploring_bdk_rn.md

```diff
@@ -0,0 +1,655 @@
+
+```javascript
+  const [syncResponse, setSyncResponse] = useState();
+  const [address, setAddress] = useState();
+  const [transaction, setTransaction] = useState();
+  const [recipient, setRecepient] = useState('');
+  const [amount, setAmount] = useState();
+```
+
+A click event handler for send button, we will use the  [`quickSend()`](https://github.com/LtbLightning/bdk-rn#quicksend) method to send specified amount in sats to  address.
+
+```javascript
+	const sendTx = async () => {
+    const { data } = await BdkRn.quickSend({
+      address: recipient,
+      amount: amount,
+    });
+    setTransaction(data);
+    setDisplayText(JSON.stringify(data));
+  };
+```
+
+ Add a new section for send transaction functionality. We will need an input box for receiver address and an input box for amount to send. We will also need a button to trigger the transaction.
+
+Let's add the send transaction section and UI components below `{/* input boxes and send transaction button */}`
+
+```jsx
+    {/* input boxes and send transaction button */}
+    <View style={styles.sendSection}>
+      <Fragment>
+        <TextInput
+          style={styles.input}
+          placeholder="Recepient Address"
+          onChangeText={setRecepient}
+        />
+        <TextInput
+          style={styles.input}
+          placeholder="Amount (in sats)"
+          onChangeText={e => setAmount(parseInt(e))}
+        />
+        <Button
+          title="Send Transaction"
+          style={styles.methodButton}
+          onPress={sendTx}
+        />
+      </Fragment>
+    </View>
+```
+
+
+We should now be able to send a transaction as long as there is sufficient balance. 
+
+<img src="./exploring_bdk_rn/bdk_rn_send.png" style="display: block; margin: 0px auto; zoom: 50%;" />
+
+
+
+## Conclusion
+
+The app we created can be distributed for both iOS and Android thus sharing a code base, the development focused on use cases and we did not have to code intricate bitcoin node, transaction and sync-related functionalities this was managed by `bdk-rn` allowing the product to focus on the functionality. This is how `bdk` and `bdk-rn` intend to make Rapid Bitcoin Application Development possible.
+
+
+
+bdk-rn intends to expose functionality and API from bdk which has a wide variety of API with granular details allowing for many interesting use cases to be implemented. bdk-rn and bdk are constantly updated and enhanced based on developer feedback both from the bitcoin community as well as bitcoin product development teams.
+
+
+
+Stay tuned for more APIs and enhancements coming to bdk-rn in the near future. Feature and API requests are most welcome. New blogs and tutorials will be published soon for a more in-depth exploration of bdk-rn. 
+In the meantime keep in touch with the project on [GitHub](https://github.com/LtbLightning/bdk-rn) and [Twitter](https://twitter.com/BitcoinZavior).
+
+
+#### References:
```

Comment:
Should we include the livestream to? - https://www.youtube.com/channel/UCUq_ZdezVWKPvkWRicAYxLA

Reply:
Yes, I have added the link.